apps/evm_family/evm_helpers.c

```diff
@@ -161,3 +212,23 @@ bool evm_derivation_path_guard(const uint32_t *path, uint32_t depth) {
 
   return false;
 }
+
+bool evm_get_msg_data_hash(const evm_sign_msg_context_t *ctx, uint8_t *digest) {
```

Comment:
change the name from `evm_get_msg_data_hash` to `evm_get_msg_data_digest` for consistency

Reply:
updated